src/mongocxx/test/spec/unified_tests/runner.cpp

```diff
@@ -29,10 +38,18 @@ using namespace mongocxx;
 using namespace bsoncxx;
 using namespace spec;
 
+using bsoncxx::builder::basic::kvp;
+using bsoncxx::builder::basic::make_document;
+
 using schema_versions_t =
     std::array<std::array<int, 3 /* major.minor.patch */>, 1 /* supported version */>;
 constexpr schema_versions_t schema_versions{{{1, 0, 0}}};
 
+spec::apm_checker& get_apm_checker() {
```

Comment:
Created an APM map that maps client IDs to APM checkers. Also created a function `init_maps` to control the maps' construction and destruction since there is a dependency between the two. The comment in that function gives more detail.